Makefile

```diff
@@ -147,18 +145,9 @@ target-%:  ## Builds the specified target defined in the Dockerfile. The build r
 
 local-%:  ## Builds the specified target defined in the Dockerfile using the local output type. The build result will be output to the specified local destination.
 	@$(MAKE) target-$* TARGET_ARGS="--output=type=local,dest=$(DEST) $(TARGET_ARGS)"
-	@PLATFORM=$(PLATFORM) DEST=$(DEST) bash -c '\
```

Comment:
I think this is still needed, as long as you do `local-` with more than a single arch.

I'm not positive in fact we should remove `BUILDKIT_MULTI_PLATFORM`

Reply:
this restores the old behavior where each arch has a folder (buildkit default). 

> I'm not positive in fact we should remove BUILDKIT_MULTI_PLATFORM

For local target this has no effect

Reply:
> this restores the old behavior where each arch has a folder (buildkit default).

but Talos for example doesn't do this, and instead combines things removing `linux_amd64` folders

Reply:
so before the change `kres` was closer to Talos `Dockerfile` - not sure what is the proper in the end

we can probably make this step optional if we want to, but this produces correct result when building several binaries for multiple arches